mne/report.py

```diff
@@ -553,7 +553,6 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
     {{endif}}
     {{if comments is not None}}
         <br><br>
-        <div class="comments">
```

Comment:
@teonlamont : I see a matching `<div>` to the one you removed. Indentation is critical to identifying the matching divs.